cmd/ipfs/daemon.go

```diff
@@ -338,6 +339,11 @@ func daemonFunc(req cmds.Request, res cmds.Response) {
 		res.SetError(err, cmds.ErrNormal)
 		return
 	}
+	if mount && offline {
+		res.SetError(errors.New("mount is not supported in offline mode"),
```

Comment:
not *currently* supported.

This is a usecase we do want, we just don't have time to have it in for the release